rst2pdf/styles.py

```diff
@@ -610,7 +611,11 @@ def readStyle(self, ssname):
         fname = self.findStyle(ssname)
         if fname:
             try:
-                return rson_loads(open(fname).read())
+                # Is it an older rson/json stylesheet with .style extension?
+                if fname[-5:] == "style":
```

Comment:
Could we use `os.path.splitext` to get the extension instead of this? This was trigger for e.g. `foo.mystyle` too.

Reply:
I couldn't figure out the "Python" way for this, thanks for the tip!